src/ui/MainWindow.cpp

```diff
@@ -213,10 +213,19 @@ namespace wfl::ui
         }
     }
 
+    void MainWindow::onNotificationReceived(bool attention)
```

Comment:
Please change the function signature to `onNotificationChanged` since the signal notifies the status of notification (whether in a notified state or not)